config/runtime.exs

```diff
@@ -171,6 +177,7 @@ config :mindwendel, :options,
     ),
   feature_file_upload: feature_file_upload,
   feature_brainstorming_removal_after_days: delete_brainstormings_after_days,
+  feature_privacy_imprint_enabled: feature_privacy_imprint_enabled,
```

Comment:
I'd usually suggest grouping the feature flags together under a key, grouping them. The key can then also be handled by a module that wraps it and the access. I.e. I usually have a module like `FeatureFlag.enabled?(flag_name)`